scripts/test_zeth_cli

```diff
@@ -67,6 +67,15 @@ zeth sync
 note_id=`zeth ls-notes | tail -n 1 | grep -oe '^[A-Za-z0-9]\+'`
 ! [ "" == ${note_id} ]
 charlie_pk=`cat ../charlie/zeth-address.pub`
+
+# Ensure that bob has been able to generate an address and sync from the
+# blockchain without querying the prover_server for the configuration.
+
+if [ -e prover-config.cache ] ; then
+    echo Expected no prover config file
+    exit 1
+fi
```

Comment:
What is the issue if the file exists? If it's not needed, then ignore. Why fail?

Reply:
This code tests that we were able to perform some basic operations (generate a key, set up a network config) without querying the prover server.